doc/topics/the-event-loop-timers-and-nexttick.md

```diff
@@ -0,0 +1,183 @@
+# Overview of the Event Loop, Timers, and `process.nextTick()`
+
+The Following diagram shows a simplified overview of the event loop's
+order of operations.
+
+       ┌───────────────────────┐
+    ┌─>│        timers         │
+    │  └──────────┬────────────┘
+    │  ┌──────────┴────────────┐
+    │  │   pending callbacks   │
+    │  └──────────┬────────────┘      ┌───────────────┐
+    │  ┌──────────┴────────────┐      │   incoming:   │
+    │  │         poll          │<─────│  connections, │
+    │  └──────────┬────────────┘      │   data, etc.  │
+    │  ┌──────────┴────────────┐      └───────────────┘
+    └──│     setImmediate      │
+       └───────────────────────┘
+
+*note: each box will be referred to as a "phase" of the event loop.*
+
+*There is a slight discrepancy between the Windows and the Unix/Linux
+implementation, but that's not important for this demonstration.  The most
+important parts are here.  There are actually seven or eight steps, but the
+ones we care about — ones that Node.js actually uses are these four.*
+
+## timers
+
+This phase executes callbacks scheduled by `setTimeout()` and `setInterval()`.
+
+When you create a timer, you make a call to `setTimeout()`.  Then, when
+the poll phase of the event loop is entered, the number of ms before the 
+soonest timer is to be called is set as the poll's timeout. Meaning the 
+poll phase will return after "timeout" ms.  After that many 
+milliseconds, the `poll` phase will return and wrap back around to the 
+timers phase where those callbacks can be processed.
+
+Take note that the poll phase can only return while idle; execution of 
```

Comment:
'can only return' is kind of confusing, IMO.  I think you mean to say that node.js can only enter the poll phase when otherwise idle?  (Which is mostly correct, although libuv can and will poll for I/O without blocking, i.e., poll with a zero millisecond timeout.)